src/App.js

```diff
@@ -19,6 +25,11 @@ function App() {
           </Route>
           <Route path="/support" element={<Support />} />
           <Route path="/energy-analytic" element={<EnergyAnalytic />} />
+          <Route path="/" element={<SignUp />} />
+          <Route path="signin" element={<SignIn />} />
+          <Route path="verification/code" element={<Verification />} />
```

Comment:
We only have one verification page so I think removing the `/code` isn't a bad idea. what do you think?
```suggestion
          <Route path="verification/code" element={<Verification />} />
          <Route path="verification" element={<Verification />} />
```